slsDetectorServers/mythen3DetectorServer/slsDetectorFunctionList.c

```diff
@@ -58,11 +61,59 @@ void basictests() {
     return;
 #else
 	// faking it
-    firmware_check_done = 1;
+    //firmware_check_done = 1;
```

Comment:
Please remove the comments faking it and firmware_check_done, as you do the check now